utils/model_uploader/trigger_ml_models_release.sh

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Will the changes to GitHub Actions workflow come in later PR? Wanted to see how this script is being used.
Thanks!

Reply:
It's at the end of these two scripts:
- `.github/workflows/model_uploader.yml` in PR #209 
- `.github/workflows/model_listing_uploader.yml` in PR #210